views/scss/qtiCreator/_outcome-editor.scss

```
@@ -79,4 +79,9 @@
             }
         }
     }
+}
+
+// For solve issue in Chrome with scroll: https://stackoverflow.com/questions/22898335/strange-behavior-of-overflow-auto-on-chrome
+.item-editor-sidebar #sidebar-right-response-properties {
```

Does it really needed to nest an ID selector under another one?
```suggestion
#sidebar-right-response-properties {
```